source/module_hamilt/hamiltlcao.cpp

```diff
@@ -0,0 +1,125 @@
+#include "hamiltlcao.h"
+#include "module_base/global_variable.h"
+#include "module_base/timer.h"
+
+#include "src_lcao/global_fp.h"
+#include "src_pw/global.h"
+
+namespace ModuleHamilt
+{
+
+template<typename T, typename T1>
+void HamiltLCAO<T,T1>::constructFixedReal()
```

Comment:
I think defining member functions under one `template<typename T, typename T1>
void HamiltLCAO<T,T1>` is more concise.

Reply:
I tried this suggestion, but it doesn't work, any more detail?

Reply:
> I tried this suggestion, but it doesn't work, any more detail?

No ideas... Sorry